var/spack/repos/builtin/packages/r-genomicalignments/package.py

```diff
@@ -34,19 +34,27 @@ class RGenomicalignments(RPackage):
     homepage = "https://bioconductor.org/packages/GenomicAlignments/"
     git      = "https://git.bioconductor.org/packages/GenomicAlignments.git"
 
+    version('1.16.0', commit='db032a459e5cf05a2a5c2059662a541827112974')
     version('1.14.2', commit='57b0b35d8b36069d4d94af86af051f0129b28eef')
     version('1.12.2', commit='b5d6f19e4a89b6c1c3e9e58e5ea4eb13870874ef')
 
     depends_on('r-biocgenerics@0.15.3:', type=('build', 'run'))
     depends_on('r-s4vectors@0.13.13:', type=('build', 'run'))
-    depends_on('r-iranges@2.5.36:', type=('build', 'run'), when='@1.12.2')
-    depends_on('r-iranges@2.11.16:', type=('build', 'run'), when='@1.14.2')
-    depends_on('r-genomeinfodb@1.11.5:', type=('build', 'run'), when='@1.12.2')
-    depends_on('r-genomeinfodb@1.13.1:', type=('build', 'run'), when='@1.14.2')
-    depends_on('r-genomicranges@1.27.19:', type=('build', 'run'), when='@1.12.2')
-    depends_on('r-genomicranges@1.29.14:', type=('build', 'run'), when='@1.14.2')
+    depends_on('r-s4vectors@0.17.28:', when='@1.16.0', type=('build', 'run'))
+    depends_on('r-iranges@2.5.36:', when='@1.12.2', type=('build', 'run'))
+    depends_on('r-iranges@2.11.16:', when='@1.14.2', type=('build', 'run'))
+    depends_on('r-iranges@2.13.25:', when='@1.16.0', type=('build', 'run'))
```

Comment:
these should probably be open ranges

Reply:
bioconductor packages don't have 'in between' versions that are ever released. 1.12.2 is part of bioconductor 3.5, 1.14.2 is 3.6, and 1.16.0 is 3.7.  They never release a 1.15 (to the public)